docs/modules/ROOT/pages/changelog.adoc

```diff
@@ -1,5 +1,19 @@
 = Changelog
 
+[[release-2024-09-11]]
+== September 11th, 2024
+
+* *General*
+** Feature: Update onboarding dialog based on the updated design.
+** Feature: Parallelize requests to improve initial load times.
+** Fix: Bug in feature quota enforcement banner.
+** Fix: Bug in CSS imports.
+** Fix: Add backend validation for multisig networks.
+** Fix: Show different error if user is already in the org.
+** Fix: Premature input validation.
+** Fix: Improve error messages when using expired or used activation link.
+** Fix: Avoid conflicting params with existing params.
+
```

Comment:
@collins-w I still find quite a few things missing from [what was done](https://linear.app/openzeppelin-development/view/current-release-candidate-a89238e90fa2b). 
Not everything there should go to the changelog but probably more than what's in this one.